### PR TITLE
packages/cli: make src-relative imports work

### PR DESCRIPTION
**Pull request by Rugvip**
Makes src-relative imports work for the cli, more description in the comments.

Partial fix for #210

### CODE REVIEW

**Comment by Rugvip** [comment]
Looked into using rollup for building the cli instead of the amd hack, but not gonna continue with that for now. The rollup config would need to be customized to run in a NodeJS environment, which would increase complexity of that config quite a bit, only to support the cli build. Another issue is that rollup didn't support lazy-loading using require out of the box, which breaks #485, and I'm not sure how tricky it is to fix.

File: packages/cli/bin/backstage-cli [hunk 1]
```diff
@@ -5,12 +5,60 @@ const path = require('path');
 // Figure out whether we're running inside the backstage repo or as an installed dependency
```
Comment (marcuseide) [inline packages/cli/bin/backstage-cli:5]: Add the Copyright `notice` in this file too?

**Review by marcuseide — APPROVED** [review]
Can't say I understand everything... But 👍

File: packages/cli/bin/backstage-cli (hunk 1 above)
Comment (Rugvip) [reply packages/cli/bin/backstage-cli:5]: Yup :p

Comment (Rugvip) [reply packages/cli/bin/backstage-cli:5]: #494

**Pull request merged**